auto-resource-redux/src/LogisticManager.lua

```diff
@@ -105,10 +105,12 @@ local function handle_items_request(storage, force, entity, item_requests)
   local entity_position = entity.position
   local nets = entity.surface.find_logistic_networks_by_construction_area(entity_position, force)
   local chests = {}
+  local avail_net
```

Comment:
Is the order of the found networks deterministic? What happens in the case of overlapping networks? I'm not sure if I'd want to prioritise players over vehicles, for example.

Reply:
I don't know what would happen with overlapping mobile logistic networks.  I suppose it wouldn't much matter if both have an available construction robot.